scripts/prow/cli-integration-release-upgrade.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+apk add nodejs npm
+kyma_get_previous_release_version_return_version=$(curl --silent --fail --show-error -H "Authorization: token ${BOT_GITHUB_TOKEN}" "https://api.github.com/repos/kyma-project/kyma/releases" | jq -r 'del( .[] | select( (.prerelease == true) or (.draft == true) )) | sort_by(.tag_name | split(".") | map(tonumber)) | .[-2].target_commitish | split("/") | .[-1]')
```

Comment:
Do you need to call GitHub API to get latest version? Kyma is tagged semantically and actually you have the complete information about Git state once it's cloned.

It'd be beneficial if you could drop usage of GitHub token and rely on native git calls.